src/components/views/Causes.js

```diff
@@ -15,9 +15,8 @@ class Causes extends Component {
 
   removeCause(id){
     console.log('click', id);
-    socket.emit("causes::remove", id, { cascade: true }, (error, cause) => {
-      console.log('Remove a cause', cause);
-    })
+    const causes = feathersClient.service('/causes');
```

Comment:
wasn't sure about the best react place to put this so left it here for now close to .remove function that uses it